.env.test

```diff
@@ -0,0 +1,29 @@
+PORT=3001
+DB_HOST=postgres
+DB_PORT=5432
+DB_USERNAME=scraperapi
+DB_PASSWORD=scraperapipassword
+DB_DATABASE_NAME=scraperapi
+REDIS_HOST=redis
+REDIS_PORT=6379
+REDIS_PASSWORD=password
+WEB3_NODE_URL_1=https://mainnet.infura.io/v3/
+WEB3_NODE_URL_10=https://optimism-mainnet.infura.io/v3/
+WEB3_NODE_URL_288=https://boba-mainnet.gateway.pokt.network/v1/lb/
+WEB3_NODE_URL_42161=https://arbitrum-mainnet.infura.io/v3/
+WEB3_NODE_URL_137=https://polygon-mainnet.infura.io/v3/
+REFERRAL_DELIMITER_START_TIMESTAMP=1657290720
+ENABLE_SPOKE_POOLS_EVENTS_PROCESSING=false
+ENABLE_REFERRALS_MATERIALIZED_VIEW_REFRESH=false
+# specify the strategy used for updating sticky referrals. Valid options: queue | cron | disable
+STICKY_REFERRAL_ADDRESSES_MECHANISM=queue
+DISABLE_CRONS=true
+# Following distances in blocks to guarantee finality on each chain. Format should be a map chainId -> blocks.
+# E.g. { "137": 100 }
+FOLLOWING_DISTANCES=
+JWT_SECRET=secret
+# DISCORD_CLIENT_ID and DISCORD_CLIENT_SECRET are the credentials of the Discord app and they are obtained from https://discord.com/developers/applications.
+DISCORD_CLIENT_ID=clientId
+DISCORD_CLIENT_SECRET=clientSecret
+# the url accessed after the Discord authorization processed is fulfilled
+DISCORD_REDIRECT_URI=http://localhost
```

Comment:
I decided to check this file into the repository because it doesn't contain any sensitive information and allows anyone to checkout the repo and run the e2e tests directly locally. Makes sense to you @amateima?